### PR TITLE
Illumio - Fix Invalid IP List HREF

### PR DESCRIPTION
**Pull request by dsommerville-illumio**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues

No related issues.

## Description

Fixes hardcoded IP list HREF in Rapid Ransomware Containment playbook.

Re-submitting this PR after previous branch desync from upstream.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Illumio/ReleaseNotes/1_0_18.md
Comment (YuvHayun) [inline Packs/Illumio/ReleaseNotes/1_0_18.md:12]:
```suggestion
- Fixed an issue where the playbook included a hardcoded IP list HREF.
```

**Review by YuvHayun — CHANGES_REQUESTED** [review]
Hey @dsommerville-illumio, thx for your contribution!
The PR looks good.
Please fix the pre-commit & validation failures.

**Comment by dsommerville-illumio** [comment]
> Hey @dsommerville-illumio, thx for your contribution! The PR looks good. Please fix the pre-commit & validation failures.

@YuvHayun - The pre-commit error is 

```
Error: Playbook Rapid ransomware containment - Illumio uses inputs that do not appear in the inputs section: Answers
```

which seems to be triggering on the scriptarguments root for input values taken from a previous collection task. The `Enter the necessary inputs.Answers` root with accessor was correct in 6.2.0, has this formatting changed since then?

Note also that this error doesn't come up when running `demisto-sdk validate -i Packs/Illumio` locally (version 1.25.1), is there a difference between that and the pre-commit validation?

**Comment by YuvHayun** [comment]
Hey @

> > Hey @dsommerville-illumio, thx for your contribution! The PR looks good. Please fix the pre-commit & validation failures.
> 
> @YuvHayun - The pre-commit error is
> 
> ```
> Error: Playbook Rapid ransomware containment - Illumio uses inputs that do not appear in the inputs section: Answers
> ```
> 
> which seems to be triggering on the scriptarguments root for input values taken from a previous collection task. The `Enter the necessary inputs.Answers` root with accessor was correct in 6.2.0, has this formatting changed since then?
> 
> Note also that this error doesn't come up when running `demisto-sdk validate -i Packs/Illumio` locally (version 1.25.1), is there a difference between that and the pre-commit validation?

The error doesn't come up when you run it locally because this particular validation is running only when triggered with -g.
That also might be the reason why we didn't encounter it so far.
The fix should be pretty simple and we can take a look at it together if you wish.
Also, please make sure to run `./Utils/git_pull_master_into_fork.sh` as your branch seems to be outdated.

**Comment by dsommerville-illumio** [comment]
> The error doesn't come up when you run it locally because this particular validation is running only when triggered with -g. That also might be the reason why we didn't encounter it so far. The fix should be pretty simple and we can take a look at it together if you wish. Also, please make sure to run `./Utils/git_pull_master_into_fork.sh` as your branch seems to be outdated.

@YuvHayun - sounds good, can you please recommend a fix for the issue or point me to the relevant documentation?

**Comment by YuvHayun** [comment]
@yashgaikwad-crest can you sign the from please?
Triggering tests at the mean time.

**Review by melamedbn — APPROVED** [review]

**Comment by YuvHayun** [comment]
Tests passed, waiting for @yashgaikwad-crest to sign before merging to internal pr.

**Comment by dsommerville-illumio** [comment]
@YuvHayun - @yashgaikwad-crest is no longer working on this pack, he made the initial change but may not be able to sign the agreement. Can you waive his signature on the CLA here?

**Comment by YuvHayun** [comment]
Hey @dsommerville-illumio 
In that case we have 3 options:
- you can try and ask him to see if he might be able to do so.
- you can try [squash his commit](https://stackoverflow.com/questions/5189560/how-do-i-squash-my-last-n-commits-together)
- you can try re-open a new pr after closing this one and making sure you're up to date with master.

Let me know which you choose.

**Comment by YuvHayun** [comment]
@dsommerville-illumio thx for your contribution!
merging to internal pr.

**Review by YuvHayun — APPROVED** [review]

**Pull request merged**
